pages/education/school-resources.md

```diff
@@ -152,29 +153,29 @@ Required if your school has 20 or more GI Bill students
 
 </div>
 
-
-***
+---
 
 <div>
-  <h2 id="upcoming-events">Upcoming events</h2>
-  
-<b>[Arizona Veterans Program Association-2019 Fall Mini-Conference >](https://www.benefits.va.gov/gibill/resources/education_resources/school_certifying_officials/conferences_and_events.asp)</b>
 
-November 1, 2019 -- Mesa, AZ
+<h2 id="upcoming-events">Upcoming events</h2>
 
-<b>[Oklahoma Veterans Education Specialists Association (OVESA) Fall Conference >](https://www.benefits.va.gov/gibill/resources/education_resources/school_certifying_officials/conferences_and_events.asp)</b>
+<ul id="get" class="hub-page-link-list"> 
+  
+<li class="hub-page-link-list__item"><a href="https://www.benefits.va.gov/gibill/resources/education_resources/school_certifying_officials/conferences_and_events.asp"><b>Arizona Veterans Program Association-2019 Fall Mini-Conference ></a><br>   
```

Comment:
You need to close your bold tag inside the anchor tag, then open a new bold tag for the content. Same for the next two list items.

Reply:
Updated the bold tags